subsys/fw_metadata/fw_metadata.c

```diff
@@ -51,7 +51,8 @@ __fw_info struct fw_firmware_info m_firmware_info =
 
 void fw_abi_provide(u32_t address)
 {
-	const struct fw_firmware_info *their_fw_info = fw_firmware_info_get(address);
+	const struct fw_firmware_info *their_fw_info
```

Comment:
Is their maybe a little vague? I'm not really sure if I have a better suggestion.

provided_fw_info?